config/release/kustomization.yaml

```diff
@@ -3,5 +3,5 @@ bases:
 
 patchesStrategicMerge:
 - manager_image_patch.yaml
-apiVersion: v1beta1
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
could you please comment on this change in the commit message? or preferably move it to a separate commit?